src/cow-react/modules/limitOrders/state/limitOrdersWarningsAtom.ts

```diff
@@ -0,0 +1,19 @@
+import { atom } from 'jotai'
+
+interface LimitOrdersWarnings {
+  isRateImpactAccepted: boolean
```

Comment:
I'm confused with the concept of rateImpact

whats the difference between rateImpact, and priceImpact?

Reply:
It's a very good question.

`RateImpact` - it's a difference between the market price and the entered price.

For example:
Market price: `1 ETH` = `2000 DAI`
Entered price (using UI): `1 ETH` = `1900` DAI
RateImpact = (1900 * 100 / 2000) - 100 = -5%

---

`PriceImpact` - it's a difference between the input currency amount and the output currency amount in USD equivalent. It's the same price impact as on Swap widget.
For example: 
User set limit order: `12 WETH` -> `14556 USDT`, wherein, `12 WETH` ≈ `$14,590.8` and `14556 USDT` ≈ `$14,570.56`.
PriceImpact = (14570.56 * 100 / 14590.8) - 100 = -0.13%

---

I think we need them both, because:
 - we must show to user the difference between the price he entered and the current market price
 - we must notify a user when the trade is illiquid

Reply:
I understand the importance of these 2 warnings on the one hand, but on the other hand due to too many warnings I decided to report case 2. 
Look: you have 2 cases for the price deviation, you agree, you press on the Confirm box in the Order confirm modal... and then  receive another box with the Price impact warning. Too many warnings at a time :)
![2 times confirm](https://user-images.githubusercontent.com/70885163/202668213-035b5be1-ba1f-4e20-9b2a-58fc0e12b89b.png)

Reply:
@elena-zh agree. However, it's possible to have these warnings separately.
I can add a condition and don't ask a user to accept `Price impact` warning when we already have `Rate impact` warning with a checkbox. Wdyt?

Reply:
I think it might be a good idea!

Reply:
@anxolin What do you say? :)